tests/test_qpu.py

```diff
@@ -1,200 +1,265 @@
+
+        As the results coming from the qvm are stochastic, a constraint of  3 out of 5 runs was added.
+        """
+
+        device = np.random.choice(VALID_QPU_LATTICES)
+        dev_qpu = qml.device('forest.qpu', device=device, load_qc=False, readout_error=[0.9, 0.75],
+                            symmetrize_readout="exhaustive", calibrate_readout="plus-eig", shots=QVM_SHOTS)
+
+        @qml.qnode(dev_qpu)
+        def circuit():
+            qml.RY(np.pi/2, wires=[0])
+            qml.CNOT(wires=[0, 1])
+            return qml.expval(qml.PauliZ(0))
+
+        assert np.allclose(circuit(), 0.0, atol=2e-2)
+
+    @flaky(max_runs=5, min_passes=3)
+    def test_2q_gate_pauliz_identity_tensor(self):
+        """Test that the PauliZ tensor Identity observable works correctly.
+
+        As the results coming from the qvm are stochastic, a constraint of  3 out of 5 runs was added.
+        """
+        device = np.random.choice(VALID_QPU_LATTICES)
+        dev_qpu = qml.device('forest.qpu', device=device, load_qc=False, readout_error=[0.9, 0.75],
+                            symmetrize_readout="exhaustive", calibrate_readout="plus-eig", shots=QVM_SHOTS)
+
+        @qml.qnode(dev_qpu)
+        def circuit():
+            qml.RY(np.pi/2, wires=[0])
+            qml.CNOT(wires=[0, 1])
+            return qml.expval(qml.PauliZ(0) @ qml.Identity(1))
+
+        assert np.allclose(circuit(), 0.0, atol=2e-2)
+
+    @flaky(max_runs=5, min_passes=3)
+    @pytest.mark.parametrize("a", np.linspace(-0.5, 2, 6))
+    def test_2q_gate_pauliz_pauliz_tensor(self, a):
+        """Test that the PauliZ tensor PauliZ observable works correctly.
+
+        As the results coming from the qvm are stochastic, a constraint of  3 out of 5 runs was added.
+        """
+        device = np.random.choice(VALID_QPU_LATTICES)
+        dev_qpu = qml.device('forest.qpu', device=device, load_qc=False, readout_error=[0.9, 0.75],
+                            symmetrize_readout="exhaustive", calibrate_readout="plus-eig", shots=QVM_SHOTS)
+
+        @qml.qnode(dev_qpu)
+        def circuit(x):
+            qml.RY(x, wires=[0])
+            qml.Hadamard(wires=1)
+            qml.CNOT(wires=[0, 1])
+            return qml.expval(qml.PauliZ(0) @ qml.Identity(1))
+
+        assert np.allclose(circuit(a), np.cos(a), atol=2e-2)
+        # Check that repeated calling of the QNode works correctly
+        assert np.allclose(circuit(a), np.cos(a), atol=2e-2)
+
+    @flaky(max_runs=10, min_passes=1)
+    @pytest.mark.parametrize("a", np.linspace(-np.pi/2, 0, 3))
+    @pytest.mark.parametrize("b", np.linspace(0, np.pi/2, 3))
+    def test_2q_gate_pauliz_pauliz_tensor_parametric_compilation_off(self, a, b):
+        """Test that the PauliZ tensor PauliZ observable works correctly, when parametric compilation
+        is turned off.
+
+        As the results coming from the qvm are stochastic, a constraint of  3 out of 5 runs was added.
```

Comment:
This doesn't match the flaky decorator (which says 1/10 tests)

Reply:
Thanks for getting that!